tools/ci_build/github/azure-pipelines/orttraining-linux-nightly-ortmodule-test-pipeline.yml

```diff
@@ -0,0 +1,28 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_Nightly_ORTModule_tests
+
+  timeoutInMinutes: 120
+  pool: 'Onnxruntime-Linux-GPU-NC6sv3'
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  # Entry point for all ortmodule training tests
+  - script: |
+      docker run \
+        --gpus all \
+        --rm \
+        --volume $(Build.SourcesDirectory)/orttraining/orttraining/test/python:/onnxruntime_src \
+        --volume $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements_torch_nightly:/requirements_torch_nightly \
+        ptebic.azurecr.io/internal/azureml/aifx/nightly-ubuntu2004-cu113-py38-torch1130dev \
```

Comment:
Considering that we are going to stop publishing cuda 11.3 package, shall we request PTCA team to give us a 11.6 package?

Reply:
Sure. But we don't need to block this PR. Let's merge this.